vcpkg.json

```
@@ -6,6 +6,10 @@
       "asio",
       "curl",
       "expat",
+      {
+        "name": "fontconfig",
+        "platform": "linux"
```

```suggestion
        "platform": "!(osx | windows)"
```

CMake and the code seem to want fontconfig for non-Apple unixy systems, so we should match that with our platform expression.

---

This matched what we had for libuuid, but I've updated it as requested.

However, I think the `$<$<PLATFORM_ID:Linux>:Fontconfig::Fontconfig>` check might still only pick this up on Linux specifically?